### ISSUE TITLE
Improve feedback if jar isn't started in unzipped directory

### ISSUE THREAD

**Issue by krichter722** [jsettlers/settlers-remake#282]
If `java -jar JSettlers.jar` is invoked outside the unpacked zip archive created in `target` a dialog (probably a `JOptionPane`) with `settlers-folder-still-invalid` is displayed and `java.io.IOException: Not a resources folder: /mnt/richtercloud/sources/settlers-remake/resources in /mnt/richtercloud/sources/settlers-remake` is logged to console which makes it possible to guess what's to do, but still not as helpful as it could be.

experienced with v0.2-alpha-107-g418f5f6

**Comment by andreas-eberle** [comment]
@codingberlin: With the new UI, you could improve the error reporting when the resources folder isn't found.

**Comment by andreasb242** [comment]
I would like to create something like a wizard, which shows the user what he has to do to bring up JSettlers working.
What do you think?

![check-dialog](https://cloud.githubusercontent.com/assets/822958/12372148/510cbd3a-bc4d-11e5-86e7-24ce8b04fd73.png)

**Comment by andreas-eberle** [comment]
@andreasb242: I'm not sure if such a wizard would be an improvement. 

I think directly showing what's the problem when there is a problem, would be the more direct way. E.g. The user that simply extacts the zip file, like it is described in the manual, will never need to know about the possibility that it might not work when you start it from within the zip file. So there is no point in confusing him with such information.

**Comment by krichter722** [comment]
> I think directly showing what's the problem when there is a problem, would be the more direct way. E.g. The user that simply extacts the zip file, like it is described in the manual, will never need to know about the possibility that it might not work when you start it from within the zip file. So there is no point in confusing him with such information. 

While I agree that there no _need_ for such information, it doesn't hurt to provide it and don't understand what could be confusing about it. Furthermore, I - and that's very extreme and no one has to agree (the next 10 years ;)) - think that everytime someone has to look into a manual if a problem could be handled by software feedback a design error is involved.

**Comment by andreas-eberle** [comment]
@krichter722: If agree with your position on the manuals. But that is also exactly, why I would not show the user any information he will never need. In the example picture above, this information would be the "green ticks" (and the text beside them). If something is not working, I, as a user, don't want to know the parts that are working... 

It doesn't make me much happier to see I managed to get around most of the obstacles if I still failed. So when I failed, tell me how to fix it. (IMHO)

**Comment by krichter722** [comment]
@andreas-eberle Agreed. Displaying a dialog with a useful error message (and instructions if possible) if and only if the jar is invoked in a directory where it can't resolve resource pathes relatively it a better solution.
